arviz/plots/autocorrplot.py

```diff
@@ -110,6 +111,7 @@ def plot_autocorr(
     )
 
     if ax is None:
+        show = backend_kwargs.pop("show", None) if backend_kwargs is not None else None
```

Comment:
I agree with @OriolAbril .All this show logic is more confusing to me. Can we just default show in the backend_kwargs and avoid all these conditionals and nones?

Reply:
No, because mol does not have show and this could be None.